### PR TITLE
config: adds git diff to github workflow

### PR DESCRIPTION
**Pull request by rnveach**
Adding git diff to ensure there are no differences after running install.

Expecting CI to fail currently without fix.

### CODE REVIEW

**Comment by rnveach** [comment]
Github Workflow reported expected differences now: https://github.com/checkstyle/eclipse-cs/actions/runs/4097913763/jobs/7066638067#step:6:111

I was hoping other 2 would complete to ensure they could run the bash script, but GH cancelled them after the first failed.

**Comment by rnveach** [comment]
https://github.com/checkstyle/eclipse-cs/actions/runs/4097947074/jobs/7066695091
All workflows are fixed with new commit.

File: docs/dtds/checkstyle-metadata_1_0.dtd
Comment (rnveach) [inline docs/dtds/checkstyle-metadata_1_0.dtd:7]: @Calixte When we discussed `docs` at https://github.com/checkstyle/eclipse-cs/pull/457#discussion_r1091318388 , it was my understanding that `docs` was not updated real-time with the snapshots and is only updated during a release. Did I misunderstand something?

CI at https://github.com/checkstyle/eclipse-cs/pull/460 didn't run travis, so this is why it was missed.
CI at https://github.com/checkstyle/eclipse-cs/pull/457 did run travis and it reported no issues, but `docs` was not required to be updated.

How come my PR did not updated `docs` but yours did? Is it related to the change you made in [net.sf.eclipsecs.doc/build.properties](https://github.com/checkstyle/eclipse-cs/pull/460/files#diff-c1a23c35875bfc9d7e2032a9e48486c40759e09ec9e01fe19898b9da04f9b835) ?

**Review by romani — APPROVED** [review]
ok to merge, looks like spacing issues.

we need green CI.

**Comment by romani** [comment]
@Calixte , please review this PR to let us come to green state and ease all other PRs.

**Comment by rnveach** [comment]
Yes, it is spacing issues from a previous fix but they shouldn't have occurred from previous conversations and previous CI runs. I think something else is wrong and waiting for confirmation from @Calixte unless @Bananeweizen may know.

**Comment by romani** [comment]
merging this infra changes to unblock other CI updated.

File: docs/dtds/checkstyle-metadata_1_0.dtd
Comment (Calixte) [reply docs/dtds/checkstyle-metadata_1_0.dtd:7]: I assume so, yes. The docs directory will get some changes from time to time, when maven runs. I just never commit them, because the release process does it.

**Pull request merged**
